Cargo.toml

```diff
@@ -90,3 +92,73 @@ members = [
 exclude = [
 	"wireguard-go-rust-wrapper"
 ]
+
+[workspace.dependencies]
+anyhow = "1"
```

Comment:
I personally feel like it would make sense to only leave library dependencies here. As it we only thing we truly ship. 
So something like clap, and anyhow, that is only build by clis, would not be added here.

But I don't have to strong of an opinion.

Reply:
As far as I can tell, specifying a dependency in that section doesn't change the resulting artifacts. If it did, I would agree with you, but since it doesn't, I think the simplified maintenance makes it worth it